libs/components/packages/src/schematics/migrations/update-8/update-polyfill/update-polyfill.schematic.ts

```diff
@@ -13,21 +13,50 @@ function removePolyfillCode(
   project: ProjectDefinition
 ): Rule {
   return async (tree): Promise<Rule | void> => {
-    const targetsToUpdate: string[] = [];
+    // Saves a list of files that need to be checked for SKY UX polyfills.
+    const polyfillsFiles = new Set<string>();
+
+    // Saves a list of targets that use each polyfills file.
+    const targetsWithPolyfills = new Map<string, Set<string>>();
```

Comment:
This is needed because the `test` target references two "polyfills" files: `src/polyfills.ts` and `src/test.ts`. The resulting `Map` looks like this:
```
[
  ['src/polyfills.ts', ['build', 'test']],
  ['src/test.ts', ['test']]
]
```